.circleci/config.yml

```diff
@@ -326,13 +326,16 @@ jobs:
 
   run-test-ios-17:
     <<: *base-job
+    resource_class: macos.m1.medium.gen1
```

Comment:
Xcode 15 is very slow on CircleCI (https://discuss.circleci.com/t/severe-performance-problems-with-xcode-15/49205/11). This makes it a bit more bearable.